odoo/addons/base/models/ir_cron.py

```diff
@@ -64,11 +70,11 @@ class ir_cron(models.Model):
                                       ('days', 'Days'),
                                       ('weeks', 'Weeks'),
                                       ('months', 'Months')], string='Interval Unit', default='months')
-    numbercall = fields.Integer(string='Number of Calls', default=1, help='How many times the method is called,\na negative number indicates no limit.')
-    doall = fields.Boolean(string='Repeat Missed', help="Specify if missed occurrences should be executed when the server restarts.")
     nextcall = fields.Datetime(string='Next Execution Date', required=True, default=fields.Datetime.now, help="Next planned execution date for this job.")
     lastcall = fields.Datetime(string='Last Execution Date', help="Previous time the cron ran successfully, provided to the job through the context on the `lastcall` key")
     priority = fields.Integer(default=5, help='The priority of the job, as an integer: 0 means higher priority, 10 means lower priority.')
+    failures = fields.Integer(default=0, help='The number of consecutive failures of this job')
```

Comment:
I don't like plural on field names (it gives me `attachment.datas` PTSD :scream_cat:), without reading the help message my intuition tells me it is the total failures count.
```suggestion
    consecutive_failures_count = fields.Integer(string="Consecutive failures", default=0, help="How many times in a row did this job failed due to an error. It resets every time the job succeeds.")
```

Reply:
can we add `, reset on success.` like in the other help?